src/config_web.py

```diff
@@ -45,7 +46,9 @@
         (r"/xsrf_token", "xsrf.XSRFToken"),
         (r"/logout", "web.handlers.login.LogoutHandler"),
         (r"/login/github", "web.handlers.auth.GitHubLoginHandler"),
-        (r"/login/orcid", "web.handlers.auth.ORCIDLoginHandler")
+        (r"/login/orcid", "web.handlers.auth.ORCIDLoginHandler"),
+        (r"/{ver}/metadata/?", "web.handlers.metadata.MyGenesetMetadataSourceHandler"),
+        (r"/metadata/?", "web.handlers.metadata.MyGenesetMetadataSourceHandler"),
         ]
```

Comment:
I believe the default `APP_LIST` includes both metadata handler routings with default metadata handlers. Although what you have should still work, but it's better to remove or update the default metadata handlers, rather than just appending two new metadata routings.

Reply:
I tried not adding the /metadata routes to the `APP_LIST`. The `/metadata` endpoint worked. However, didn't show the new stats fields.

Reply:
I just meant to add a line like this before this block to remove the default metadata routings:
```
APP_LIST = [x for x in APP_LIST if x[0].endswith("metadata/?")]
```

Or you can also do this instead:
```
For handler in APP_LIST:
    if handler[0].endswith("metadata/?"):
         handler[1] = "web.handlers.metadata.MyGenesetMetadataSourceHandler"
```
The reason is [the default APP_LIST defined here](https://github.com/biothings/biothings.api/blob/bc3d07f4502b98cfabcbe2dfebd842c1e43a6a58/biothings/web/settings/default.py#L72-L86) include those metadata handlers:

Reply:
It was changed. Because it is a list of tuples, had to convert the tuple to a list before changing the value. After, I converted it to a tuple.